packages/create-webpack-app/src/utils/logger.ts

```diff
@@ -1,16 +1,16 @@
 import { WebpackCLILogger } from "webpack-cli";
 import { green, yellow, Color, red, cyan, cyanBright, blue, blueBright } from "colorette";
 import { PlopActionHooksChanges, PlopActionHooksFailures } from "../types";
-import { basename } from "path";
+import { relative } from "path";
 
 const prefix: string = blueBright("create-webpack");
 const getLogger = (): WebpackCLILogger => {
```

Comment:
Can't we use the logger like below?

```js
import webpackCLI from "webpack-cli"

const cli = new webpackCLI();

cli.logger()
```

Reply:
I think that I am using logger only and defining it is easier than importing the whole webpackCLI().
also earlier I tried doing it but was getting into a lot of dependency complexities.

So this method was easier.

Reply:
Also I told you last time that the issue was that webpack-cli doesnt exports cli, only definitions.
```console
error| '"webpack-cli"' has no exported member named 'webpackCLI'. Did you mean 'IWebpackCLI'?
```

Reply:
We should avoid code duplication wherever possible